llvm/include/llvm/Analysis/DXILResource.h

```diff
@@ -463,7 +463,7 @@ class DXILResourceMap {
   /// ambiguous so multiple creation instructions may be returned. The resulting
   /// ResourceInfo can be used to depuplicate unique handles that
   /// reference the same resource
-  SmallVector<dxil::ResourceInfo> findByUse(const Value *Key) const;
+  SmallVector<dxil::ResourceInfo *> findByUse(const Value *Key);
```

Comment:
Now that setting the value is internal to the analysis, it might make sense to make this an internal/static function? Feels a bit strange to leave the non-const function in the public api.

Thoughts?

Reply:
Why is it no longer const?